ensenso_camera/src/pose_utilities.cpp

```diff
@@ -73,22 +78,33 @@ void writePoseToNxLib(tf2::Transform const& pose, NxLibItem const& node)
   }
 }
 
+double fixZeroValue(double value, double epsilon = 0.00001)
```

Comment:
Ist irgendwie ein bisschen unsympathisch, das immer zu machen. Ich nehme an, das war für den Vergleich mit Identity? In dem Fall sollte die Toleranz beim Vergleichen gehandelt werden, aber eigentlich sollte das an der Stelle keine numerischen Probleme geben.